code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1152,13 +1153,13 @@
 	if((temp_turf.z != our_turf.z) || M.stat!=CONSCIOUS) //Not on the same zlevel as us or they're dead.
 //		to_chat(world, "[(temp_turf.z != our_turf.z) ? "not on the same zlevel as [M]" : "[M] is not concious"]")
 		if(temp_turf.z != 2)
-			to_chat(src, "The mind of [M] is too faint...")//Prevent "The mind of Admin is too faint..."
-
-
+			if(prob(5))//Can't tell who is missing as easily as before. I feel like this is a fun alternative to removing it altogether
```

Comment:
People with psy resist don't show up on the list of possible entities to
psy, so there would be no reason to show if our message was sent or not. I
didn't change much outside of how the unreachable list after you press the
button is generated. I feel like it'll be more confusing if I put a "psy
ping detected no resisting entities".

Reminder, the point of peer review on a PR is to recieve player feedback.
Just gunna put that out there.